README.md

```diff
@@ -32,24 +38,90 @@ the list you should be able to find _Collabora Online_ and enable it.
 
 From there you can access the module specific configuration.
 
-### Development installation
+Please check the "Configuration" section below!
+
+
+Development / demo installation
+-------------------------------
 
-If you intend to develop or contribute to the module, you can install
-directly from git.
+A local demo and testing instance can be installed using docker-compose.
 
-In your Drupal setup, in the directory `modules/contrib` extract the
-module into a directory `collabora_online`.
+### Requirements
 
-You can get it directly with git:
+- [Docker](https://www.docker.com/get-docker)
+- [Docker Compose](https://docs.docker.com/compose/)
+
+### Installation steps
+
+First, git clone the repository into a new directory, outside of any other Drupal project.
 
 ```sh
 git clone https://github.com/CollaboraOnline/collabora-drupal.git collabora_online
 ```
+or
+
+```sh
+git clone git@git.drupal.org:project/collabora_online.git
```

Comment:
really we should stick to the github repo as it will always have the latest code. And the releases are tagged anyway

Otherwise the https URL should be used by default:
 
```suggestion
git clone https://git.drupalcode.org/project/collabora_online.git
```

Reply:
I removed the Drupal git url.
If it ever becomes more relevant, we can change the README.